launch/client.py

```diff
@@ -538,7 +538,7 @@ def create_model_endpoint(
         gpus: int = 0,
         min_workers: int = 1,
         max_workers: int = 1,
-        per_worker: int = 1,
+        per_worker: int = 10,
```

Comment:
https://en.wikipedia.org/wiki/Anchoring_(cognitive_bias) lmao

Reply:
Is there a bias for picking the number `5`, given that it's still on one hand, and also still typable with the left hand? lol